keyrings/alt/file.py

```diff
@@ -78,9 +81,9 @@ def priority(self):
             __import__('Crypto.Cipher.AES')
             __import__('Crypto.Protocol.KDF')
             __import__('Crypto.Random')
-        except ImportError:
+        except ImportError:     # pragma: no cover
```

Comment:
Looks like you can [use mock on sys.modules](http://stackoverflow.com/a/31718136/70170) if you want to make sure it's covered.